### PR TITLE
Excludes development-only files from the npm package

### PR DESCRIPTION
**Pull request by shinnn**
Currently the npm package of this module includes some development-only files, for example `test.js` and `README.hbs`.

This PR excludes them from the package and reduce file size.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,9 @@
   "license": "MIT",
   "description": "Convert a bytes (and octets) value to a more human-readable format. Choose between metric or IEC units.",
   "repository": "https://github.com/75lb/byte-size.git",
+  "files": [
+    "index.js"
```
Comment (shinnn) [inline package.json]: These files are automatically included.

* `LICENSE`
* `README.md`
* `package.json`

**Comment by shinnn** [comment]
**Before**

```
$ du -ach node_modules/byte-size
8.0K	node_modules/byte-size/.DS_Store
4.0K	node_modules/byte-size/.npmignore
4.0K	node_modules/byte-size/.travis.yml
8.0K	node_modules/byte-size/index.js
4.0K	node_modules/byte-size/LICENSE
4.0K	node_modules/byte-size/package.json
4.0K	node_modules/byte-size/README.hbs
4.0K	node_modules/byte-size/README.md
8.0K	node_modules/byte-size/test.js
 32K	node_modules/byte-size/yarn.lock
 80K	node_modules/byte-size
 80K	total
```

**After**

```
$ du -ach node_modules/byte-size
8.0K	node_modules/byte-size2/index.js
4.0K	node_modules/byte-size2/LICENSE
4.0K	node_modules/byte-size2/package.json
4.0K	node_modules/byte-size2/README.md
 20K	node_modules/byte-size2
 20K	total
```

About 60 KB disk usage will be saved by this change.

**Comment by 75lb** [comment]
released in v4.0.2, thanks 👍 🇯🇵

**Comment by shinnn** [comment]
@75lb No problem. Thank *you* for creating useful modules!

I recommend that you apply the same change to your other projects to reduce bundle size.

**Pull request merged**